### PR TITLE
release 2.4.0

### LINKED ISSUE #223: release 2.4.0

### PR DESCRIPTION
**Pull request by cofyc**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #223 

**Special notes for your reviewer**:

**Release note**:
```
NONE
```

### CODE REVIEW

File: helm/provisioner/values.yaml [hunk 1]
```diff
@@ -106,7 +106,10 @@ daemonset:
   #
   # Defines Provisioner's image name including container registry.
   #
-  image: quay.io/external_storage/local-volume-provisioner:v2.3.4
+  # Since 2.4.0 we push multi-arch image to gcr.io/k8s-staging-sig-storage/local-volume-provisioner.
+  # Use it when you deploy in non-amd64 environment.
+  #
+  image: quay.io/external_storage/local-volume-provisioner:v2.4.0
```
Comment (msau42) [inline helm/provisioner/values.yaml:112]: Should we use k8s.gcr.io/sig-storage registry instead? (We'll need to promote the staging image once we tag)

Comment (cofyc) [reply helm/provisioner/values.yaml]: pushed a new commit https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/229/commits/46da21477d8590007ad1b9bd102e245b83bf1d9b PTAL

**Comment by cofyc** [comment]
once the 2.4.0 tag is pushed, I'll run `generate.sh` script to promote the image (https://github.com/kubernetes/k8s.io/pull/1456)

**Pull request merged**